hooks/useChat.tsx

```diff
@@ -78,6 +106,7 @@ const useChat = () => {
       id: uuidV4(),
     });
     handleAiChatSubmit(e);
+    await goToNewConversation(input);
   };
 
   return {
```

Comment:
This hook is too long. Please create smaller, more modular, hooks.
- actual: hook is longer than 100 lines of code.
- expected: all files are less than 100 lines of code.
- recommendation: put your new changes in a new hook rather than creating new function definitions in the same hook.